CHANGELOG.md

```diff
@@ -37,8 +36,39 @@
 - ReactComponentFactoryProxy no longer `implements Function`
     - This should not be a breakage, since as of Dart 2.0 inheriting from Function has had no effect
 
-#### Potential behavior breakages
-- Component and Component2 members `props`/`state`/`jsThis` are late, will now throw instead of being null if accessed before initialized (e.g., in a constructor, final class field, or static lifecycle method).
+#### Behavior breakages unlikely to cause issues
+- Component and Component2 members `props`/`state`/`jsThis` are late, will now throw instead of being null if accessed before initialized.
```

Comment:
#nit might be worth linking to the Dart documentation for `late` here?

Reply:
Good idea. Added in https://github.com/Workiva/react-dart/pull/373/commits/cb5cfd60ee956f41d8a89144e19d704ce7c160f8